oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/UserInfoHandler.java

```diff
@@ -66,6 +66,7 @@ public UserInfoHandler(ConfigurationService configurationService) {
                 new UserInfoService(
                         new DynamoService(configurationService),
                         new DynamoIdentityService(configurationService),
+                        new DynamoClientService(configurationService),
```

Comment:
Are there any terraform changes needed to give this lambda permission to use this table?

Reply:
It has the `dynamo_client_registry_read_access_policy` attached to it which should be adequate